core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -522,8 +522,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     RemoveOtherReligions("Removes other religions when spreading religion", UniqueTarget.Unit),
 
     CanActionSeveralTimes("Can [action] [amount] times", UniqueTarget.Unit),
-    // TODO needs to be more general
-    BonusForUnitsInRadius("Bonus for units in 2 tile radius 15%", UniqueTarget.Unit),
+    BonusForUnitsInRadius("Bonus for units in [amount] tile radius [relativeAmount]%", UniqueTarget.Unit),
```

Comment:
If you change the unique anyway, I'd make it more like the other Strength bonus uniques at the same time:
`[realtiveAmount]% Strength for all units in [amount] tiles`
Possibly you could even turn the `all` into a parameter, so `[relativeAmount]% Strength for [mapUnitFilter] units in [amount] tiles`

Reply:
... see my version. may even be verbatim